azure-pipelines-v3.yml

```diff
@@ -75,19 +75,19 @@ jobs:
       DOCS_OPS_TOKEN: $(DOCS_OPS_TOKEN)
       CODECOV_TOKEN: $(CODECOV_TOKEN)
 
-  # Push to sandbox MyGet feed
+  # Push to sandbox vsts feed
   - task: NuGetCommand@2
     condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/v3'))
     displayName: Push to Sandbox MyGet Feed
     inputs:
       command: push
       packagesToPush: drop/**/*.nupkg
-      nuGetFeedType: external
-      publishFeedCredentials: myget.docfx-v3-sandbox
+      publishVstsFeed: 'docfx/docs-build-v3-test'
```

Comment:
We tend to put all public packages to `docs-public-packages` feed: `<add key="doc-public-packages" value="https://docfx.pkgs.visualstudio.com/docfx/_packaging/docs-public-packages/nuget/v3/index.json" />`

Also update README.md to point to the new feed